docs/aws.md

```diff
@@ -1,39 +1,112 @@
+
+**Note**: that with AWS Route53, you can have subdomains in a single hosted zone, so you can have `cluster1.testclusters.mydomain.com` under `mydomain.com`.
+
+2) Pick a DNS name under this zone to be the name of your cluster.  kops will set up DNS so your cluster can be reached on this name.  For example, if your zone was `mydomain.com`, a good name would be `kubernetes.mydomain.com`, or `dev.k8s.mydomain.com`, or even `dev.k8s.myproject.mydomain.com`. We'll call this `NAME`.
+
+3) Kops uses the [Official AWS Go SDK](https://github.com/aws/aws-sdk-go). You will need to set up your AWS credentials as defined [here](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html)
+
+4) Pick an S3 bucket that you'll use to store your cluster configuration - this is called your state store.  You can `export KOPS_STATE_STORE=s3://<mystatestorebucket>` and then kops will use this location by default.  We suggest putting this in your bash profile or similar.  A single registry can hold multiple clusters, and it can also be shared amongst your ops team (which is much easier than passing around kubecfg files!)
+
+
+## Create your first cluster
+
+Before we create a cluster, we need to generate a `cluster config`
+
+In these examples we assume you have already exported `KOPS_STATE_STORE`, otherwise you will need to append each command with `--state s3://<mystatestorebucket>`
+
 ```
-${GOPATH}/bin/kops update cluster ${NAME} --yes
+kops create cluster --zones=us-east-1c ${NAME}
 ```
 
-If you have problems, please set `--v=8` and open an issue, and ping justinsb on slack!
+Notice in this example we defined a single AWS Availability Zone. These represent the nodes in your cluster, that are typically recommended to run each in their own Availability Zone. Best practices dictate uses more than one here.
+
+We can customize our cluster by editing our `cluster config`
+
+```
+kops edit cluster ${NAME}
+```
+
+This will open up the cluster manifest YAML file in your favorite text editor. Here is where a user can define very specific parts of their cluster. For now, lets leave this as default.
+
+
+Lets go ahead and create the cluster in AWS!
+
+```
+kops update cluster ${NAME} --yes
+```
+
+Think of the `--yes` flag as a way of saying "*Yes! I am very sure I want to create this cluster, and I understand it will cost me money*". You will notice this flag in a few other places as well.
+
+## Accessing your cluster
+
+A friendly reminder that kops runs asynchronously, and it will take your cluster a few minutes to come online.
+
+Remember when you installed `kubectl` earlier? The configuration for your cluster was automatically generated and written to `~/.kube/config` for you!
+
+A simple Kubernetes API call can be used to check if the API is online and listening. Let's use `kubectl`
+
+```
+kubectl get nodes
+```
+
```

Comment:
We can have them do a kubectl install busybox or something. We should link k8s tutorial as well.